geograypher/entrypoints/render_labels.py

```diff
@@ -52,8 +53,10 @@ def render_labels(
             See TexturedPhotogrammetryMesh.load_texture
         render_savefolder (PATH_TYPE):
             Where to save the rendered labels
-        transform_file (typing.Union[PATH_TYPE, None], optional):
-            File containing the transform from local coordinates to EPSG:4978. Defaults to None.
+        input_CRS: (pyproj.CRS):
+            The coordinate reference system to interpret the mesh data in. Since meshes are not
```

Comment:
Could this be clarified a bit? I think what this means as "Read in the mesh file and assume that `input_CRS` is what the data is stored in", but reading the docstring I could interpret it as "Read that mesh file and convert it to `input_CRS` before doing anything else`.

Reply:
You're interpretation is correct, I'll clarify that.